tool_water_balance/tools/waterbalance.py

```diff
@@ -1037,8 +1037,8 @@ def get_missing_agg_vars(self):
         return missing_vars
 
     def run(self):
-        selected_ds = self.ts_datasource.rows[0].datasource()
-        if not selected_ds.ds_aggregation:
+        selected_datasource = self.ts_datasources.rows[0].datasource()
+        if not selected_datasource.ds_aggregation:
```

Comment:
Here we call it `selected_datasource` and in other places we call it `active_datasource`. Is there a difference?

Reply:
Goeie.
Er zijn zoveel naam-variaties...